### PR TITLE
Pass EXTRA_ARGS to run.sh (fixes #2714)

### LINKED ISSUE #2714: EXTRA_ARGS should be passed to run.sh in finalExec
Current:

https://github.com/itzg/docker-minecraft-server/blob/3d2b88effde08b470ea84d929302c0bb7e3f7fb7/scripts/start-finalExec#L297

Suggested:

```sh
  exec mc-server-runner "${mcServerRunnerArgs[@]}" --shell bash "${SERVER} ${EXTRA_ARGS}"
```

Use case:
- Allow setting `--universe ./worlds` and other flags.

The `run.sh` script is documented as follows and supports passing extra program args:

```sh
# cat run.sh
#!/usr/bin/env sh
# Forge requires a configured set of both JVM and program arguments.
# Add custom JVM arguments to the user_jvm_args.txt
# Add custom program arguments {such as nogui} to this file in the next line before the "$@" or
#  pass them to this script directly
java @user_jvm_args.txt @libraries/net/minecraftforge/forge/1.19.2-43.3.5/unix_args.txt "$@"
# 
```

### PR DESCRIPTION
**Pull request by robinbraemer**

### CODE REVIEW

**Comment by robinbraemer** [comment]
I'm not a bash pro so we will probably want to specifically want the variables to expand to args the same way as done in finalArgs.

**Comment by robinbraemer** [comment]
Confirming the added DEBUG_EXEC also prints as expected

```
[init] Using Forge supplied run.sh script...
+ exec mc-server-runner --stop-duration 60s --shell bash run.sh '--universe ./worlds/'
```

**Comment by itzg** [comment]
I'm glad to hear using multiple args seems to have worked. That would have been my only doubt also.

File: scripts/start-finalExec [hunk 1]
```diff
@@ -294,7 +294,10 @@ elif [[ $SERVER =~ run.sh ]]; then
     echo "SETUP_ONLY: bash ${SERVER}"
     exit
   fi
-  exec mc-server-runner "${mcServerRunnerArgs[@]}" --shell bash "${SERVER}"
+  if isTrue "${DEBUG_EXEC}"; then
+    set -x
+  fi
+  exec mc-server-runner "${mcServerRunnerArgs[@]}" --shell bash "${SERVER}" "${EXTRA_ARGS}"
```
Comment (itzg) [inline scripts/start-finalExec:300]: and just saw your follow up. Removing the double quotes around the car should help it here. 

Sorry if I merged your PR too soon. Will need a new one for the tweak.

**Comment by robinbraemer** [comment]
Trying out `"${EXTRA_ARGS[@]}"` would behave...

**Comment by robinbraemer** [comment]
Better reopen this PR if possible, trying to fix it here

**Comment by itzg** [comment]
> Trying out `"${EXTRA_ARGS[@]}"` would behave...

I don't think adding the `@` will work since it's a string variable and not an array.

**Comment by itzg** [comment]
Notice it's not quoted when added here

https://github.com/itzg/docker-minecraft-server/blob/3d2b88effde08b470ea84d929302c0bb7e3f7fb7/scripts/start-finalExec#L323

**Comment by itzg** [comment]
We'll need a new PR since merged PRs can't be re-opened.

**Pull request merged**